media-api/test/lib/ElasticSearchTest.scala

```diff
@@ -58,37 +106,112 @@ class ElasticSearchTest extends FunSpec with BeforeAndAfterAll with Matchers wit
     }
   }
 
+  describe("syndicationStatus query on the Syndication tier") {
+    it("should return 0 results if a Syndication tier queries for SentForSyndication images") {
+      val search = SearchParams(tier = Syndication, syndicationStatus = Some(SentForSyndication))
+      val searchResult = ES.search(search)
+      whenReady(searchResult, timeout, interval) { result =>
+        result.total shouldBe 0
+      }
+    }
+
+    it("should return 3 results if a Syndication tier queries for QueuedForSyndication images") {
+      val search = SearchParams(tier = Syndication, syndicationStatus = Some(QueuedForSyndication))
+      val searchResult = ES.search(search)
+      whenReady(searchResult, timeout, interval) { result =>
+        result.total shouldBe 3
+
+        val imageIds = result.hits.map(_._1)
+        imageIds.size shouldBe 3
+        imageIds.contains("test-image-1") shouldBe true
+        imageIds.contains("test-image-2") shouldBe true
+        imageIds.contains("test-image-4") shouldBe true
+      }
+    }
+
+    it("should return 0 results if a Syndication tier queries for BlockedForSyndication images") {
+      val search = SearchParams(tier = Syndication, syndicationStatus = Some(BlockedForSyndication))
+      val searchResult = ES.search(search)
+      whenReady(searchResult, timeout, interval) { result =>
+        result.total shouldBe 0
+      }
+    }
+
+    it("should return 0 results if a Syndication tier queries for AwaitingReviewForSyndication images") {
+      val search = SearchParams(tier = Syndication, syndicationStatus = Some(AwaitingReviewForSyndication))
+      val searchResult = ES.search(search)
+      whenReady(searchResult, timeout, interval) { result =>
+        result.total shouldBe 0
+      }
+    }
+  }
+
+  describe("syndicationStatus query on the internal tier") {
+    it("should return 1 image if an Internal tier queries for SentForSyndication images") {
+      val search = SearchParams(tier = Internal, syndicationStatus = Some(SentForSyndication))
+      val searchResult = ES.search(search)
+      whenReady(searchResult, timeout, interval) { result =>
+        result.total shouldBe 1
+      }
+    }
+
+    it("should return 3 images if an Internal tier queries for QueuedForSyndication images") {
+      val search = SearchParams(tier = Internal, syndicationStatus = Some(QueuedForSyndication))
+      val searchResult = ES.search(search)
+      whenReady(searchResult, timeout, interval) { result =>
+        result.total shouldBe 3
+      }
+    }
+
+    it("should return 3 images if an Internal tier queries for BlockedForSyndication images") {
+      val search = SearchParams(tier = Internal, syndicationStatus = Some(BlockedForSyndication))
+      val searchResult = ES.search(search)
+      whenReady(searchResult, timeout, interval) { result =>
+        result.total shouldBe 3
+      }
+    }
+
+    it("should return 0 images if an Internal tier queries for AwaitingReviewForSyndication images") {
```

Comment:
mismatch of description and assertion :)

Reply:
Oops! Fixed.